ui/appui-react/src/appui-react/statusfields/toolassistance/ToolAssistanceField.tsx

```diff
@@ -541,15 +523,25 @@ export class ToolAssistanceField extends React.Component<
           >
             {dialogContent}
           </ToolAssistanceDialog>
-        </StatusBar.Popup>
-      </>
+        }
+      >
+        <Button
+          styleType="borderless"
+          startIcon={<>{toolIcon}</>}
+          className={classnames("temp", this.props.className)}
```

Comment:
@FlyersPh9 I have some questions about `ToolAsisstanceField` styling. 
1. First of all, it has a little triangle at the bottom right corner which I suppose symbolizes that it is expandable. I see the same concept used in toolbars. However, I was wondering why this is the only component in `StatusBar` that uses this triangle even though many other are also button that open a popup e.g. `MessageCenterField`, `SnapModeField`. Do we want to keep it then?
2. Should this field have a constant size (like it is now)? I suppose this was done so the other `StatusBar` fields do not move when changing the active tool (since names of tools are of different length).
3. Also this field had some additional styling for smaller screens e.g. the width of this field would decrease at smaller screens and the text would wrap into multiple lines. I would like to get rid of the wrapping because all other field components have text without wrapping so that would be just consistent. Not too sure about the width getting smaller on smaller screens though. What do you think?
![image](https://github.com/iTwin/appui/assets/138560995/e63d1466-aab8-42fb-bacf-a3c9bb015078)

Reply:
For the 2nd question: I think we should have a consistent width to prevent content shift.  I don't know how that width is set currently, but we should set the width of the label to use the `ch` unit.

For the 1st and 3rd, can we take a look in a call?